src/tests.js

```diff
@@ -0,0 +1,32 @@
+const itParam = require('mocha-param');
+const { expect } = require('chai');
+const { odometer, odometerReverse } = require("./index");
+
+describe("Odometer function", function() {
+  const cases = [
+    {input: [0, 0, 0, 0], output: [0, 0, 0, 1]},
+    {input: [4, 3, 9, 5], output: [4, 3, 9, 6]},
+    {input: [4, 3, 4, 9], output: [4, 3, 5, 0]},
+    {input: [4, 8, 9, 9], output: [4, 9, 0, 0]},
+    {input: [4, 9, 9, 9], output: [5, 0, 0, 0]},
+    {input: [9, 9, 9, 9], output: [0, 0, 0, 0]},
```

Comment:
Help me understand why this does not match the problem statement:
https://github.com/NewBoCo/Code-Challenge/blob/590397a897c2c197bf02993247366736b392b561/README.md#L24-L28

Reply:
@dahlbyk Because he asked about it and we realized it made no sense for it to go from 9999 to 1000. So he was told 0000 was valid because an odometer would do the same if it reached it's limit. Craig and I forgot to fill you in on that discussion.

Reply:
I probably should have added a note here. I reached out to @tektechnologies (in slack) and a asked him if `[0,0,0,0]` made more sense given the odometer analogy. He did agree that the case I made makes sense. If you would like I can modify for this to return the original expected value. If that is the case should I also do the same for the reverse method?

Reply:
I agree with the decision to diverge from the "requirements". A note would have been good.